roles/vm_infra/tasks/rename_resize_boot_disk.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: "Get the '{{ current_vm.name }}' VM info"
+  ovirt_vm_info:
+    auth: "{{ ovirt_auth }}"
+    pattern: "name={{ current_vm.name }}"
+    fetch_nested: true
+    nested_attributes:
+      - "bootable"
+    all_content: true
+  register: current_vm_info
+
+- name: "Get the '{{ current_vm.name }}' VM boot disk ID"
+  ansible.builtin.set_fact:
+    current_vm_boot_disk_id: "{{ item.id }}"
+  loop: "{{ current_vm_info.ovirt_vms[0].disk_attachments }} "
+  when: item.bootable is defined and item.bootable
+
+- name: "Get the '{{ current_vm.name }}' VM boot disk info"
+  ovirt_disk_info:
+    auth: "{{ ovirt_auth }}"
+    pattern: "id={{ current_vm_boot_disk_id }}"
+    fetch_nested: true
+  register: current_vm_boot_disk_info
+  when: current_vm_boot_disk_id is defined
+
+- name: "Get the Storage Domain info from the '{{ current_vm.name }}' VM boot disk"
+  ovirt_storage_domain_info:
+    auth: "{{ ovirt_auth }}"
+    id: "{{ current_vm_boot_disk_info.ovirt_disks[0].storage_domains[0].id }}"
```

Comment:
Not sure if the module accepts the `id` you might need to use the `pattern` with the id filter. And it returns a list of storage domains.

Reply:
You are absolutely right and that's why I was using `ovirt_storage_domain` in the first place, it's not possible to retrieve the information of a storage domain with `ovirt_storage_domain_info` if you only have the storage domain id :(

Reply:
resolved offline